src/components/note/note.test.js

```diff
@@ -166,7 +125,7 @@ context("Tests for Note component", () => {
       const callback = cy.stub();
 
       CypressMountWithProviders(
-        <NoteComponent onLinkAdded={callback} text="https://carbon.s" />
+        <NoteComponent onLinkAdded={callback} text="https://carbon.sage.com" />
```

Comment:
why this is changed?

Reply:
Actually I haven't find any difference after adding the valid URL but I'll take a look again here. Thank you!